README.rst

```diff
@@ -33,26 +33,11 @@ steps:
 
    .. code-block:: python
 
-      from django.conf.urls import url
-
-      from oscarapi.app import application as api
-
-      urlpatterns = (
-          # all the things you already have
-          url(r'^oscarapi/', api.urls),
-      )
-
-   or with the newer Django 2.x way
-
-   .. code-block:: python
-
-      from django.urls import path
-
-      from oscarapi.app import application as api
+      from django.urls import include
 
       urlpatterns = (
           # all the things you already have
-          path('oscarapi/', api.urls),
+          path("api/", include("oscarapi.urls")),
```

Comment:
Is it now nolonger possible to disable some functionality that is not needed?

Reply:
i used to do that by filtering urls in the app config.

Reply:
I see now that `OscarConfigMixin` also has some other url related methods as well, so I should use the app for returning the url's indeed.

Reply:
For now you need to create your own `urlpatterns` if you want this changed. But this was the case  also in the old situation.

Agreed to make it simpler like this?